custom_components/tuya_local/devices/wzm100w_presence_sensor.yaml

```diff
@@ -0,0 +1,111 @@
+name: mmWave presence sensor
+products:
+  - id: wcd9lxpvhfvqx1qw
+    name: WZ-M100-W
```

Comment:
The model number should not be used as name, just let it inherit the top level name above.